src/components/segmented-control/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {InteractiveColor} from '@atb/theme/colors';
+import {View, ViewStyle, StyleProp, TouchableOpacity} from 'react-native';
+import {StyleSheet, useTheme} from '@atb/theme';
+import ThemeText from '../text';
+
+export type SegmentOptions = {
+  onPress: () => void;
+  text: string;
+  subtext?: string;
+  selected?: boolean;
+  accessibilityHint?: string;
+};
+
+type SegmentedControlProps = {
+  color: InteractiveColor;
+  activeIndex: number;
+  options: SegmentOptions[];
+  enabled?: boolean;
+  style?: StyleProp<ViewStyle>;
+};
+
+export default function SegmentedControl({
```

Comment:
Great work on creating this reusable component! Should it maybe be named something with "radio" and each option have `accessibilityRole: 'radio'`? 🤔

Reply:
- accessibilityRole: can do!
- New naming: Suggestions? "Segmented controls" is whats used in figma. "RadioSegments" maybe?

Reply:
Yeah. RadioSegments is good :+1: